### PR TITLE
New conda-forge recipe for conneciton_pool

### PR DESCRIPTION
**Pull request by jmeppley**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
N/A
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jmeppley** [comment]
Hello, @conda-forge/help-python, can I get a review on a proposed new package?

File: recipes/connection_pool/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{% set name = "ConnectionPool" %}
+{% set author = "zhouyl" %}
+{% set version = "0.0.3" %}
+{% set sha256 = "3acb9bbd6ef9a6ca9d5e056c0afd14dbc5ca8d610d788b162412f6235a6e3759" %}
+
+package:
+  name: connection_pool
```
Comment (scopatz) [inline recipes/connection_pool/meta.yaml:7]: Why add the underscore?

**Comment by jmeppley** [comment]
That's the name of an earlier version in pip of the same package.

https://pypi.org/project/connection_pool/

Also, the name of the import.

File: recipes/connection_pool/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+    fn: {{ name }}.{{ version }}.tar.gz
+    url: https://github.com/{{ author }}/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
```
Comment (chrisburr) [inline recipes/connection_pool/meta.yaml:13]: Can we get this from PyPI instead?
```suggestion
    url: https://pypi.io/packages/source/c/{{ name }}/{{ name }}-{{ version }}.tar.gz
    sha256: bf429e7aef65921c69b4ed48f3d48d3eac1383b05d2df91884705842d974d0dc
```

File: recipes/connection_pool/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{% set name = "ConnectionPool" %}
+{% set author = "zhouyl" %}
+{% set version = "0.0.3" %}
+{% set sha256 = "3acb9bbd6ef9a6ca9d5e056c0afd14dbc5ca8d610d788b162412f6235a6e3759" %}
```
Comment (chrisburr) [inline recipes/connection_pool/meta.yaml:4]:
```suggestion
{% set name = "connection_pool" %}
{% set version = "0.0.3" %}
```

File: recipes/connection_pool/meta.yaml (hunk 1 above)
Comment (chrisburr) [inline recipes/connection_pool/meta.yaml:7]:
```suggestion
  name: {{ name|lower }}
```

File: recipes/connection_pool/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - connection_pool
+
+about:
+  home: https://github.com/{{ author }}/{{ name }}
```
Comment (chrisburr) [inline recipes/connection_pool/meta.yaml:32]:
```suggestion
  home: https://github.com/zhouyl/ConnectionPool
```

File: recipes/connection_pool/meta.yaml (hunk 2 above)
Comment (jmeppley) [reply recipes/connection_pool/meta.yaml:11]: Sure

File: recipes/connection_pool/meta.yaml (hunk 1 above)
Comment (jmeppley) [reply recipes/connection_pool/meta.yaml]: It's the name of the import and the pip package

**Comment by jmeppley** [comment]
Hello, @conda-forge/help-python, thanks for the help so far. I think I've ironed out all the wrinkles. Is this ready to be merged?

**Pull request merged**